packages/calcite-components/src/components/carousel-item/carousel-item.scss

```diff
@@ -0,0 +1,14 @@
+:host {
+  @apply flex;
+}
+
+:host([selected]) .container {
```

Comment:
Let's move forward with this since there's already precedent. We can discuss hidden-by-default rules and make any adjustments in 3.x or 4.x. Impact might be low since item/child components aren't expected to be used standalone.